src/main/kotlin/ui/taskManagementView/TaskDescriptionEditionView.kt

```diff
@@ -13,7 +14,11 @@ class TaskDescriptionEditionView(
 
     fun editDescription(task: Task) {
```

Comment:
```suggestion
    fun editDescriptionProject(task: Task) {
```
or find readable name

Reply:
the view name is enough to make it readable in my opinion

Reply:
also it is task description, not project

Reply:
So you said that, I thought it description of project from first time :::)